src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -3039,31 +2937,31 @@ namespace HVACMultiSpeedHeatPump {
         OnOffAirFlowRatio = 0.0;
         SupHeaterLoad = 0.0;
 
-        auto &MSHeatPump = state.dataHVACMultiSpdHP->MSHeatPump;
+        auto &MSHeatPump = state.dataHVACMultiSpdHP->MSHeatPump(MSHeatPumpNum);
```

Comment:
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:3042]:(style),[constVariable],Variable 'MSHeatPump' can be declared as reference to const